docker-compose.yml

```
@@ -160,3 +160,21 @@ services:
     depends_on:
       - handshake
       - handshake-api
+
+  synckeys:
+    build:
+      context: docker/synckeys
+      dockerfile: Dockerfile
+    container_name: synckeys
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/keys:/keys
+      - ./docker/data/sia:/sia-data
```

```suggestion
      - ./docker/data/sia/apipassword:/sia-data/apipassword:ro
```

mount just the apipassword file instead of the whole sia-data directory (nginx container does that)

---

Nice, will do.